base_geoengine/geo_model.py

```diff
@@ -178,7 +178,8 @@ def get_edit_info_for_geo_column(self, cursor, uid, column, context=None):
         res['edit_raster'] = raster_obj.read(
             cursor, uid, raster_id[0], context=context)
         res['geo_type'] = field.geo_type
-        res['srid'] = field.srid
+        # get srid from _columns as srid is wrong in _fields
```

Comment:
I think that this code will not work if your field is a related. In this case, `_columns .get(column)` is `None` since a related field in the new API has no column associated to the field.